spec/presenters/two_factor_authentication/webauthn_platform_selection_presenter_spec.rb

```diff
@@ -1,18 +1,41 @@
 require 'rails_helper'
 
 describe TwoFactorAuthentication::WebauthnPlatformSelectionPresenter do
-  let(:subject) { described_class.new(configuration) }
+  let(:user_without_mfa) { create(:user) }
+  let(:user_with_mfa) { create(:user) }
   let(:configuration) {}
+  let(:presenter_without_mfa) {
+ described_class.new(configuration: configuration, user: user_without_mfa)
+  }
+  let(:presenter_with_mfa) {
+ described_class.new(configuration: configuration, user: user_with_mfa)
+  }
```

Comment:
let's fix the indentation here. Also use `do...end` for multiline blocks
```suggestion
  let(:presenter_without_mfa) do
    described_class.new(configuration: configuration, user: user_without_mfa)
  end
  let(:presenter_with_mfa) do
    described_class.new(configuration: configuration, user: user_with_mfa)
  end
```